src/test/librbd/journal/test_Replay.cc

```diff
@@ -171,7 +181,11 @@ TEST_F(TestJournalReplay, AioDiscardEvent) {
   int64_t current_entry;
   get_journal_commit_position(ictx, &current_tag, &current_entry);
   ASSERT_EQ(initial_tag + 1, current_tag);
-  ASSERT_EQ(0, current_entry);
+  if (is_rbd_rwl_enabled()) {
```

Comment:
Nit: same comment. Why would RWL impact these tests? If RWL is incompatible w/ journalling, just ensure RWL is disabled.

Reply:
RWL cache is disabled when journaling is enabled. This is done in src/librbd/cache/rwl/InitRequest.cc